app/src/main/java/com/keylesspalace/tusky/fragment/TimelineFragment.java

```diff
@@ -1313,6 +1317,11 @@ private void handleBookmarkEvent(@NonNull BookmarkEvent bookmarkEvent) {
         setBookmarkForStatus(pos, status, bookmarkEvent.getBookmark());
     }
 
+    private void handleMuteConversationEvent(@NonNull MuteConversationEvent event) {
+        // TODO: Is there anything to do here but a full refresh?
```

Comment:
Is there a way to find all posts that are part of a thread and remove them without refreshing the whole timeline or doing a context request? I don't think so, you could remove the comment.

Reply:
I couldn't come up with anything either